### PR TITLE
[IOTDB-935] fix progressBar print problem

### PR DESCRIPTION
**Pull request by mychaow**

### CODE REVIEW

**Comment by samperson1997** [comment]
Hi @mychaow , many thanks for your quick fix!!! I'd like to wait and rerun the Travis test for a few times for you before giving my approval : )

**Review by samperson1997 — COMMENTED** [review]
Hi @mychaow , many thanks for your quick fix!!! I'd like to wait and rerun the Travis test for a few times for you before giving my approval : )

**Comment by HTHou** [comment]
Hi @mychaow, since you are a committer, I think you can change to use Apache repository to submit a PR now. :)

**Review by samperson1997 — APPROVED** [review]

**Pull request merged**
